cmp/tab/TabContainerModel.js

```diff
@@ -128,7 +129,8 @@ export class TabContainerModel extends HoistModel {
                         category: 'Navigation',
                         message: `Viewed ${isString(title) ? title : id} tab`,
                         // If using routing, data field specifies route for non-top-level tabs.
-                        data: route && route !== 'default' ? {route: route} : null
+                        data: route && route !== 'default' ? {route: route} : null,
+                        oncePerSession: true
```

Comment:
Could we make the 'oncePerSession' flag configurable on TCM and NavigatorModel? I think it could be useful information to track a user's entire navigation history, for traffic data on a particular tab/page or for understanding workflow that require navigation?